assets/script.js

```diff
@@ -0,0 +1,100 @@
+let library = [];
+const container = document.querySelector('.container');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const pages = document.querySelector('#pages');
+const read = document.querySelector('#read');
+// const notRead = document.querySelector('#not-read');
+// const read = document.querySelectorAll('input[name='read-check']');
```

Comment:
- [ ] Please remove these commented code lines if you need to refer to the previous versions of the code, consider using git history.